pythran/tests/test_base.py

```diff
@@ -179,6 +179,9 @@ def test_print(self):
     def test_print_tuple(self):
         self.run_test("def print_tuple(a,b,c,d): t = (a,b,c,d,'e',1.5,); print(t)", [1.,2.,3.1],3,True, "d", print_tuple=[List[float], int, bool, str])
 
+    def test_print_fstring(self):
+        self.run_test("def print_fstring(a,b,c): print(f'a: {a: 4d}; b: {b:.2f}; c: {c:s}')", 2, 6.28, "c", print_fstring=[int, float, str])
```

Comment:
I'd rather have a return than a print here, because we don't check the correctness of output. That should still validate the fstring ;-)